app/src/main/java/woowacourse/shopping/feature/BindingAdapter.kt

```diff
@@ -14,16 +14,31 @@ fun ImageView.loadImageFromUrl(url: String) {
     Glide.with(this.context).load(url).into(this)
 }
 
+@BindingAdapter("cartItems")
+fun RecyclerView.bindCartItems(items: List<Cart>?) {
+    if (adapter is CartAdapter && items != null) {
+        (adapter as CartAdapter).setItems(items)
+    }
+}
+
 @BindingAdapter("items")
-fun RecyclerView.bindItems(items: List<Goods>?) {
+fun RecyclerView.bindItems(items: List<Any>?) {
     if (adapter is GoodsAdapter && items != null) {
         (adapter as GoodsAdapter).setItems(items)
-    } else if (adapter is CartAdapter && items != null) {
-        (adapter as CartAdapter).setItems(items)
     }
 }
 
 @BindingAdapter("visible")
 fun View.setVisible(visible: Boolean) {
-    this.visibility = if (visible) View.VISIBLE else View.GONE
+    visibility = if (visible) View.VISIBLE else View.GONE
```

Comment:
제가 리뷰를 남겼는지 기억이 애매한데

```suggestion
    isVisible = visible
```

에서 `isVisible` 내부를 보셔도 좋고 저기 안에 있는 `View` 의 확장함수는 무엇이 있는지 한 번 보셔도 좋을 것 같네요

Reply:
```kotlin
public inline var View.isVisible: Boolean
    get() = visibility == View.VISIBLE
    set(value) {
        visibility = if (value) View.VISIBLE else View.GONE
    }
```

이미 View의 확장함수가 있었군요...! 들어가서 보니까 다른 확장함수들도 많네요!!
나중에 미션을 하면서 써보겠습니다 감사합니다!!

[refactor: BindingAdapter에 isVisible 확장 함수 적용](https://github.com/woowacourse/android-shopping-cart/pull/127/commits/af68917ba1b0a0686b1b865058537b09a4311f3d)